### ISSUE TITLE
About text2img generation finetuning

### ISSUE THREAD

**Issue by o0t1ng0o** [ByteDance-Seed/Bagel#294]
Hi @wangwei1237 @haooooooqi @helio9cn @TsuTikgiau @Luodian 

Thank you for releasing such an impactful work.

I try to fine-tune my text2img data on Bagel.
But I found that the generated image suffering from strange color and its shape seems correct.
I have checked the input data which is valid. I also checked the inference code that is correct.
<img width="240" height="244" alt="Image" src="https://github.com/user-attachments/assets/8dd879a8-99b8-4c1b-ac5d-d8c6e44b3044" />

What could be causing this result?

Looking forward to your reply.

**Comment by o0t1ng0o** [comment]
In addition, the MSE loss cannot converge.

<img width="278" height="210" alt="Image" src="https://github.com/user-attachments/assets/ea719497-cbce-41ca-8ee5-09550c5b303e" />
